kwc-blockly-toolbox.html

```diff
@@ -289,7 +289,7 @@
                         duration = size.height / 3;
                         this.style.overflowY = 'hidden';
                         for (let i = 0; i < buttons.length; i++) {
-                            if (over) {
+                            if (over && buttons[i].animate) {
```

Comment:
Maybe a one time check with `const canAnimate = 'animate' in HTMLElement.prototype` and then use the var